services/ui-src/src/components/forms/Form.tsx

```diff
@@ -51,14 +51,15 @@ export const Form = ({
 
   const updateRenderFields = (fields: (FormField | FormLayoutElement)[]) => {
     const updatedTargetPopulationChoices = report?.fieldData?.targetPopulations;
+    const defaultTargetPopulationCount = 4;
```

Comment:
my concern is, what happens if that list is ever altered? How would we know to update this number

Reply:
Yeah.. I feel you on this. Maybe we can get around this when we are attempting to add the other target population? Like the only reason we need the 4 is to mark whether its going to be one of the original ones or not. So if we add it during entity creation then we can remove this hard coded 4

Reply:
@ntsummers1 The default target populations have a require true property on them. I used that in the current push, the "Other" Target populations dont generate the property so that might work to distinguish them.

Reply:
Yeah, I'm cool with that!